mmcv/utils/pytorch_wrapper.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+TORCH_VERSION = torch.__version__
```

Comment:
It is repeated with [here](https://github.com/open-mmlab/mmcv/blob/7982dd1a06bbcee3eb15a56aa55b8c18ebd2bf7c/mmcv/utils/parrots_wrapper.py#L6).